manifests/ssh.pp

```diff
@@ -21,33 +21,31 @@
 # @example
 #   include profile_idds::ssh
 class profile_idds::ssh (
-  $acctd_subnets,
-  $acctd_groups,
-  $acctd_users,
-  $dev_subnets,
-  $dev_users,
-  $dev_groups,
-){
-
+  Array[String] $acctd_subnets,
+  Array[String] $acctd_groups,
+  Array[String] $acctd_users,
+  Array[String] $dev_subnets,
+  Array[String] $dev_users,
+  Array[String] $dev_groups,
```

Comment:
Did `pdk validate -a` automatically typecast these variable?

Reply:
No, pdk validate complained about it but a `-a` didn't fix it. Had to manually add them.

Are you thinking they should be a different type?

Reply:
I was just curious. No change needed.